pkg/meta/model/reorg.go

```diff
@@ -108,6 +109,20 @@ func (dm *DDLReorgMeta) SetBatchSize(batchSize int) {
 	atomic.StoreInt64(&dm.BatchSize, int64(batchSize))
 }
 
+// GetMaxWriteSpeedOrDefault gets the max write speed from DDLReorgMeta.
+// 0 means no limit.
+func (dm *DDLReorgMeta) GetMaxWriteSpeedOrDefault(defaultVal int) int {
+	if dm == nil {
```

Comment:
in which case `dm` is nil? And we should handle `nil` `dm` in `SetMaxWriteSpeed`

Reply:
I guess an old cluster without `DDLReorgMeta ` field  upgrade to a new cluster, after unmarshal, this dm can be nil

Reply:
update in https://github.com/pingcap/tidb/pull/57611/commits/7d46d9e63510985e2bbe3a92e0e7cb5056d2d5a1

Reply:
this should be handled in where it's unmarshalled in a unified way, not check in every method

Reply:
Is this the right place to handle the nil reorg meta https://github.com/pingcap/tidb/pull/57611/commits/1b25f9a3420f56783edf406d69c89e1639fe89a4